haiku-misc/neonlights/neonlights-0.0.1.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Screensaver drawing neon lines"
+DESCRIPTION="NeonLights is a screensaver that draws colorful neon lines. \
+It was based loosely on Jared Tarbell's work. Inspired by Atalax's Substrate."
+HOMEPAGE="https://github.com/pulkomandy/neonlights/"
+COPYRIGHT="2017 Adrien Destugues"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="f658b45894fe5d2349be4e06c3a0bce359985e27"
+SOURCE_URI="https://github.com/pulkomandy/neonlights/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9271f3778218bb63430da00d00b099c5d3e9d83b21d6c132533069b237199663"
+SOURCE_DIR="neonlights-$srcGitRev"
```

Comment:
@dash102, for recipes which are not using a stable release but a specific commit on a repo, we usually add a "\~git" suffix (e.g. neonlights-0.0.2\~git.recipe). But here the  srcGitRev used is only one commit ahead of the stable release, so maybe it's OK to leep the current name.
Well, let's see what @pulkomandy recommends ;-)